transaction/core/src/tx_error.rs

```diff
@@ -84,6 +84,8 @@ pub enum NewMemoError {
     Creation(MemoError),
     /// Utf-8 did not properly decode
     Utf8Decoding,
+    /// Max fee of {0} exceeded. Attempted to set fee amount: {1}
```

Comment:
```suggestion
    /// Attempted to set fee to {1} > {0} max fee
```
Maybe a little terser?

Reply:
Went with
```rust
    /// Attempted value: {1} > Max Value: {0}
```

Can modify LimitExceeded to include the numeric data points if you like that idea.